src/screens/authentication/UsernameRequest/index.js

```diff
@@ -100,10 +156,35 @@ const UsernameRequest = (props) => {
     setError(null)
   }
 
+  const onSuggestionSelected = (suggestion) => {
+    setTextInputValue(suggestion)
+    setError(false)
+  }
+
+  useEffect(() => {
+    const createSuggestions = async () => {
+      let posibleUsernames = []
+      let i = 0
+      while (i < 9) {
+        posibleUsernames[i] = (textInputValue + '_' + Math.floor((Math.random() * 100) + 1))
+        i++
+      }
+      const usedUsernames = await DatabaseService.findUsersWithUsernames(posibleUsernames)
+      if (usedUsernames) {
+        posibleUsernames = posibleUsernames.filter(u => !(usedUsernames.includes(u)))
+      }
+      setSuggestions(posibleUsernames)
+    }
+    if (usernameExists) {
```

Comment:
Add a blank line to separate the `if`

Reply:
do you separate all blocks by convention (if for return else) or only if's and return?

Reply:
All blocks.